sage/pallet/src/mock.rs

```diff
@@ -47,75 +52,57 @@ pub const BOB: MockAccountId = 2;
 pub const CHARLIE: MockAccountId = 3;
 pub const DAVE: MockAccountId = 4;
 
-pub const TREASURER: MockAccountId = 431;
+pub const TOURNAMENT_TREASURY: MockAccountId = 431;
 
 pub const SEASON_ID_0: MockSeasonId = 0;
 pub const SEASON_ID_1: MockSeasonId = 1;
 
+pub const MAIN_ASSET_ID: u32 = 0;
+pub const LOW_LIQUIDITY_ASSET_ID: u32 = 99;
+
 // Configure a mock runtime to test the pallet.
 frame_support::construct_runtime!(
 	pub struct Test {
 		System: frame_system = 0,
 		Balances: pallet_balances = 1,
-		Sage: pallet_sage = 2,
+		PalletAssets: pallet_assets = 2,
+		Sage: pallet_sage = 3,
 	}
 );
 
+#[derive_impl(frame_system::config_preludes::TestDefaultConfig)]
 impl frame_system::Config for Test {
-	type RuntimeEvent = RuntimeEvent;
-	type BaseCallFilter = frame_support::traits::Everything;
-	type BlockWeights = ();
-	type BlockLength = ();
-	type RuntimeOrigin = RuntimeOrigin;
-	type RuntimeCall = RuntimeCall;
-	type RuntimeTask = RuntimeTask;
-	type Nonce = u32;
-	type Hash = H256;
-	type Hashing = BlakeTwo256;
 	type AccountId = MockAccountId;
-	type Lookup = IdentityLookup<Self::AccountId>;
-	type Block = MockBlock;
-	type BlockHashCount = ConstU64<250>;
-	type DbWeight = ();
-	type Version = ();
-	type PalletInfo = PalletInfo;
 	type AccountData = pallet_balances::AccountData<MockBalance>;
-	type OnNewAccount = ();
-	type OnKilledAccount = ();
-	type SystemWeightInfo = ();
-	type SS58Prefix = ConstU16<42>;
-	type OnSetCode = ();
-	type MaxConsumers = frame_support::traits::ConstU32<16>;
-	type SingleBlockMigrations = ();
-	type MultiBlockMigrator = ();
-	type PreInherents = ();
-	type PostInherents = ();
-	type PostTransactions = ();
+	type Block = MockBlock;
 }
 
 parameter_types! {
 	pub const MockExistentialDeposit: MockBalance = 3;
 }
 
+#[derive_impl(pallet_balances::config_preludes::TestDefaultConfig)]
 impl pallet_balances::Config for Test {
-	type RuntimeEvent = RuntimeEvent;
-	type RuntimeHoldReason = ();
-	type RuntimeFreezeReason = ();
-	type WeightInfo = ();
-	type Balance = MockBalance;
-	type DustRemoval = ();
-	type ExistentialDeposit = MockExistentialDeposit;
 	type AccountStore = System;
-	type ReserveIdentifier = [u8; 8];
-	type FreezeIdentifier = ();
-	type MaxLocks = ();
-	type MaxReserves = ();
-	type MaxFreezes = ();
+	type ExistentialDeposit = MockExistentialDeposit;
 }
 
+#[derive_impl(pallet_assets::config_preludes::TestDefaultConfig)]
+impl pallet_assets::Config for Test {
+	type Currency = Balances;
+	type CreateOrigin = AsEnsureOriginWithArg<frame_system::EnsureSigned<MockAccountId>>;
+	type ForceOrigin = frame_system::EnsureRoot<u64>;
```

Comment:
Shouldn't this be `frame_system::EnsureRoot<MockAccountId>` ?

Reply:
Yes good spot, copy paste error from the substrate codebase 👍